config/pricealert-tg.yaml

```diff
@@ -0,0 +1,20 @@
+---
+notifications:
+  # object routing rules
+  routing:
+    trade: "$symbol"
+    order: "$symbol"
+    submitOrder: "$session" # not supported yet
+    pnL: "bbgo-pnl"
```

Comment:
do we still need this for telegram?

Reply:
can do without this
but with this the tg bot can send order notification